kernel/src/process/process/mod.rs

```diff
@@ -668,6 +686,18 @@ impl Process {
     pub fn status(&self) -> &ProcessStatus {
         &self.status
     }
+
+    // ******************* Attribute ********************
```

Comment:
Not all attributes of a process is under this part. So the more appropriate name of this section is "subreaper".

```
    // ******************* Subreaper ********************
```